README.md

```diff
@@ -1,8 +1,23 @@
 # Taarafo.Web
+[![.NET](https://github.com/hassanhabib/Taarafo.Web/actions/workflows/dotnet.yml/badge.svg)](https://github.com/hassanhabib/Taarafo.Web/actions/workflows/dotnet.yml)
 [![The Standard - COMPLIANT](https://img.shields.io/badge/The_Standard-COMPLIANT-2ea44f)](https://github.com/hassanhabib/The-Standard)
 
 This is Taarafo - an open source social media platform to allow people to clone, host and create their own social media communities, while being able to connect to other individuals and other communities around the world without fearing being censored, banned or blocked out.
 
 The way Taarafo works, is that it decenteralizes the ownership and governance of social media communities - it allows anyone, anywhere around the world to build a community to connect with their families, friends and the people they share interests with without fearing their data would be used for social or psychological research, and without fearing to be censored for their unpopular opinion.
 
 I believe everyone in the world should be able to express their opinions, it's a basic human right to be able to share your thoughts with the world fearlessly without fearing to be cancelled and completely eliminated from your cyber existence.
+
+<img width=20 src="https://www.searchmarketingaustralia.com.au/wp-content/uploads/2017/10/original_images_YouTube.png" /> [EFS015: Building Foundation Services in Blazor Apps](https://www.youtube.com/watch?v=fcRPl5IP-rs)
+<br>
+<img width=20 src="https://www.searchmarketingaustralia.com.au/wp-content/uploads/2017/10/original_images_YouTube.png" /> [EFS017: Handling Null Inputs in Foundation Services](https://www.youtube.com/watch?v=AiBhWjTZ6zU)
+<br>
+<img width=20 src="https://www.searchmarketingaustralia.com.au/wp-content/uploads/2017/10/original_images_YouTube.png" /> [EFS019: Integrating Blazor Apps w/ ASP.NET APIs](https://www.youtube.com/watch?v=nvtOKmacXKQ)
+<br>
+<img width=20 src="https://www.searchmarketingaustralia.com.au/wp-content/uploads/2017/10/original_images_YouTube.png" /> [EFS020: Implementing Blazor Foundation Services (Part 1)](https://www.youtube.com/watch?v=tHMZR3EMZJY)
+<br>
+<img width=20 src="https://www.searchmarketingaustralia.com.au/wp-content/uploads/2017/10/original_images_YouTube.png" /> [EFS022: Implementing Blazor Foundation Services (Part 2)](https://www.youtube.com/watch?v=qvwEWPyk0h8)
+<br>
```

Comment:
Missing 21.

Reply:
@ShriHumrudha there is no 21 unless it is missing from the playlist on TouTube
![image](https://user-images.githubusercontent.com/797750/141645041-f15b1e71-e23c-4a16-a71a-26e05df5d0b2.png)